dts/bindings/input/seeed,chsc6x.yaml

```diff
@@ -0,0 +1,14 @@
+description: CHSC6X touchscreen sensor
+
+compatible: "seeed,chsc6x"
```

Comment:
the vendor here should be the vendor of the actual chip, is this https://szzxv.com/static/upload/file/20220301/1646141990530969.pdf? then it should be `chipsemi,chsc6x` (you'll have to add an entry in `dts/bindings/vendor-prefixes.txt` as well)

Reply:
I couldn't find any datasheet or information for the part, so I was not sure of the constructor. I contacted Seeed Studio and asked for some more information, but they only redirected me to their Arduino library for the display: https://github.com/Seeed-Studio/Seeed_Arduino_RoundDisplay
I'll change that.

Reply:
yeah had to do some digging as well but it it seems like that's it: https://www.chipsemicorp.com/%E6%99%BA%E8%83%BD%E8%A7%A6%E6%8E%A7%E8%8A%AF%E7%89%87/

Reply:
Looks like it yeah.
I added the vendor, made the switch in the neccessary files, checked quickly that everything worked okay, and pushed.